src/input_common/udp/client.cpp

```diff
@@ -136,6 +136,9 @@ static void SocketLoop(Socket* socket) {
 
 Client::Client() {
     LOG_INFO(Input, "Udp Initialization started");
+    for (std::size_t id = 0; id < MAX_TOUCH_FINGERS; ++id) {
+        finger_id[id] = MAX_UDP_CLIENTS * 2;
+    }
```

Comment:
```suggestion
finger_id.fill(MAX_UDP_CLIENTS * 2);
```

Also consider a comment explaining the significant of `MAX_UDP_CLIENTS * 2`